junction/proposals/models.py

```diff
@@ -292,6 +292,15 @@ def get_reviewer_nick(self):
                                                           reviewer_id=self.commenter_id)
         return reviewer.nick
 
+    def get_comment_type(self):
+        if self.vote:
+            return 'Vote'
+        if self.private:
+            return 'Review'
+        if self.reviewer:
```

Comment:
This is confusing. Can you use `if/elif`. ? Check here for proper condition https://github.com/pythonindia/junction/blob/master/junction/proposals/models.py#L150